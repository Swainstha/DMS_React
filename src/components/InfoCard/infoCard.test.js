import testSnapFunction from '../../util/testSnapFunction';
import InfoCard from './index'
import React from 'react';

const dummy_data={
    data:{
        color:"yello",
        icon:"icon",
        title:"title",
        subtitle:"subtitle"
    }
}
testSnapFunction("<InfoCard/>", "Snapshot test for PhroductGridItem",<InfoCard {...dummy_data}/> )