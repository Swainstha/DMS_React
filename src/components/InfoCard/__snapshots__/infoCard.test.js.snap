// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InfoCard/> Snapshot test for PhroductGridItem 1`] = `ShallowWrapper {}`;
